src/getCookieCutterSource.js

```diff
@@ -49,6 +51,20 @@ const getCookieCutterSource = async ({ branchName, destination }) => {
     source: destinationSourceFilename,
     destination,
   });
+
+  /*
+    Code is cloned inside "frontend-cookiecutter-application-master".
+    This extracts the content files to an upper level so the code
+    will be created in the folder you indicated
+  */
+  await move(`${destination}/frontend-cookiecutter-application-master/*`, destination, { dot: true })
+    .then(() => 'Content of node_modules moved successfully!')
+    .catch((error) => {
+      console.error(error);
+    });
+
+  // Removes the temporary folder
+  rimraf.sync(`${destination}/frontend-cookiecutter-application-master`);
```

Comment:
We can use fs-extra here as well:

```suggestion
  fse.removeSync(`${destination}/frontend-cookiecutter-application-master`);
```

Reference: https://github.com/jprichardson/node-fs-extra/blob/master/docs/remove-sync.md